prepare-operator-metadata.yml

```diff
@@ -5,7 +5,7 @@
   gather_facts: false
 
   vars:
-    operator_work_dir: "/home/jenkins/agent/test-operator"
```

Comment:
I would remove this change from the patch.
I asked Scott to unify names in initial pull request. because on some places we use `/home/jenkins/agent/test-operator` and on other `home/jenkins/test-operator`. It should not case any isseu in downstream.